src/models/msg/liquidity_provider/msg_burn_tokens.ts

```diff
@@ -0,0 +1,34 @@
+import * as R from 'ramda';
+import { Categories } from '../types';
+
+class MsgBurnTokens {
+    public category: Categories;
+    public type: string;
+    public json: any;
+    public liquidityProvider: string;
+    public amount: MsgCoin[];
+
+    constructor(payload: any) {
+      this.category = 'liquidityProvider';
+      this.type = payload.type;
+      this.json = payload.json;
+      this.liquidityProvider = payload.liquidityProvider;
+      this.amount = payload.amount;
+    }
+
+    static fromJson(json: any) {
+      return new MsgBurnTokens({
+        json,
+        type: json['@type'],
+        liquidityProvider: json.liquidityProvider,
```

Comment:
`json.liquidity_provider` because 

```
message MsgBurnTokens {
  string liquidity_provider = 1 // THIS OVER HERE
      [ (gogoproto.moretags) = "yaml:\"liquidity_provider\"" ];

  repeated cosmos.base.v1beta1.Coin amount = 2 [
    (gogoproto.moretags) = "yaml:\"amount\"",
    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
    (gogoproto.nullable) = false
  ];
}
```